src/webapi/event.rs

```diff
@@ -1066,6 +1066,27 @@ reference_boilerplate! {
     convertible to ProgressRelatedEvent
 }
 
+/// An EventHandler that is called whenever the readyState attribute changes.
+/// The callback is called from the user interface thread.
+/// The XMLHttpRequest.onreadystatechange property contains the event handler to be
+/// called when the readystatechange event is fired,
+/// that is every time the readyState property of the XMLHttpRequest changes.
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/onreadystatechange)
```

Comment:
This should link to this and have the docs copied from this page:

https://developer.mozilla.org/en-US/docs/Web/Events/readystatechange